src/v2/Components/Onboarding/Steps/Genes/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { throttle } from "lodash"
 import React from "react"
 import styled from "styled-components"
-
+import { ProgressIndicator } from "../../../ProgressIndicator"
```

Comment:
Does this progress indicator ultimately map to [Palette](https://palette.artsy.net/elements/loaders/progressbar)? If not, this might be a quick win.

Reply:
Sadly no, it's this one: https://github.com/artsy/force/blob/953d162ee90f7a2223d34ddf902a1c888f0bd2f8/src/v2/Components/ProgressIndicator.tsx and I felt like it was a bridge too far to mess with it. 🤷